_posts/2020-12-04-a-new-way-to-learn-and-discover-quarkus.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: A new way to learn & discover Quarkus (even for the Quarkus veterans)
+date: 2020-12-04
+tags: extensions codestarts quickstart
+synopsis: All our tooling has been updated and can now generate Quarkus application with example code showing the true power of the selected extensions...
+author: adamevin
+---
+
+What’s nice about this new feature is that it’s useful for fresh Quarkus developers as well as experienced ones. This is because of the wide range (and growing) of Quarkus extensions, you simply can’t know it all.
+
+Would you blame me if I say that we developers are a tiny bit lazy? We also like well documented frameworks.. But reading docs is a bit boring.. Right?
+By the way you can see that Quarkus is pretty well documented here: https://quarkus.io/guides/.
+
+We developers also love to code (at least I hope you do for your own sake :)..
+So, what would you say if, when you wish to use some new extensions, to have the code showing you their true power directly ready to run in your favorite IDE? In a few seconds?
+
+All this, using your favorite buildtool (Maven, Gradle or also Gradle with Kotlin DSL) and your favorite language (Java, Kotlin or Scala).
+
+Well, it’s been slowly making its way in for a while now (since 1.7), what we in the team call “Codestarts” are now the official way of generating your Quarkus applications starting from 1.10.2.Final!
```

Comment:
```suggestion
It’s been slowly making its way in for a while now (since 1.7), what we in the team call “Codestarts” are now the official way of generating your Quarkus applications starting from 1.10!
```